### PR TITLE
beepsky no longer hates being pulled

### PR DESCRIPTION
**Pull request by cebutris**
Not being able to pull beepsky is really dumb

### CODE REVIEW

**Comment by TalkingCactus** [comment]
It should just check your ID access when you pull. 
No robotics or security = brutal beating

**Comment by cebutris** [comment]
I'll try, but it will take me like a week to figure out

**Comment by Hibou48888** [comment]
Maybe we could steal the code that the turrets use to check a person's ID for a weapons permit?

**Comment by AnonymousHybi** [comment]
i got an idea, lemme just puke something up

**Comment by AnonymousHybi** [comment]
if(!pulledby.check_access_list(_beepsky required access here_))

**Comment by Tk420634** [comment]
Is this ready to go or wat?

**Comment by AnonymousHybi** [comment]
no im gonna do some things

**Comment by Tk420634** [comment]
I'll thing you in the do somes

**Comment by Tk420634** [comment]
It's okay, Second Attempts.

**Comment by Tk420634** [comment]
(;´༎ຶД༎ຶ`)

File: code/modules/mob/living/bot/secbot.dm [hunk 1]
```diff
@@ -24,16 +24,7 @@
 
 	var/list/threat_found_sounds = list('sound/voice/bcriminal.ogg', 'sound/voice/bjustice.ogg', 'sound/voice/bfreeze.ogg')
 	var/list/preparing_arrest_sounds = list('sound/voice/bgod.ogg', 'sound/voice/biamthelaw.ogg', 'sound/voice/bsecureday.ogg', 'sound/voice/bradio.ogg', 'sound/voice/bcreep.ogg')
-//VOREStation Add - They don't like being pulled
-/mob/living/bot/secbot/Life()
-	..()
-	if(health > 0 && on && pulledby)
-		if(isliving(pulledby))
-			var/mob/living/L = pulledby
-			UnarmedAttack(L)
```
Comment (AnonymousHybi) [inline code/modules/mob/living/bot/secbot.dm:33]: do if(!L.check_access(src))
then just indent the rest of what it says. then test it in a private server.
Put it just under line 32

**Comment by AnonymousHybi** [comment]
actually just close this, i'll submit a PR to do this

**Comment by cebutris** [comment]
kk fam

**Pull request closed without merging**